react/Footer/FooterLink/FooterLink.js

```diff
@@ -36,9 +38,16 @@ FooterLink.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]),
-  linkRenderer: PropTypes.func.isRequired
+  linkRenderer: PropTypes.func.isRequired,
+  authedHref: PropTypes.string,
```

Comment:
Yep I think `authRequired` fits the bill better, I'll switch it over now. Thanks!